src/containers/App/index.js

```diff
@@ -69,42 +75,58 @@ export const AppContainer = ({ resetIncidentAction }) => {
     };
   }, [history]);
 
+  useEffect(() => {
+    // prevent continuing (and performing unncessary API calls)
+    // when the current session has not been authenticated
+    if (!isAuthenticated()) return;
+
+    if (configuration.fetchSourcesFromBackend) {
+      getSourcesAction();
```

Comment:
You can use `useDispatch` here instead of having to use `bindActionCreators` which, in turn, requires you to use `enzyme` in tests to ensure that the `getSourcesAction` function is present in the component's props. 

It doesn't make much sense to assert that a component has a specific (magically injected) prop. We do, however, need to assert that the action is called and that is what should be tested for.

Reply:
Awesome!